### PR TITLE
fix x509: certificate signed by unknown authority

### PR DESCRIPTION
**Pull request by jianzhangbjz**
Replace that base64 encoded crt with decoded one.
```yaml
! error: Import failed (InternalError): Internal error occurred: [ec2-3-138-125-3.us-east-2.compute.amazonaws.com:6001/openshifttest/busybox@sha256:c5439d7db88ab5423999530349d327b04279ad3161d7596d2126dfb5b02bfd1f: Get "https://ec2-3-138-125-3.us-east-2.compute.amazonaws.com:6001/v2/": x509: certificate signed by unknown authority, quay.io/openshifttest/busybox@sha256:c5439d7db88ab5423999530349d327b04279ad3161d7596d2126dfb5b02bfd1f: Get "https://quay.io/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)] 
```

### CODE REVIEW

**Comment by jianzhangbjz** [comment]
@kuiwang02 @jianlinliu Could you help have a review? Thanks!

File: ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh [hunk 1]
```diff
@@ -113,6 +113,22 @@ registry.redhat.io/rhel8/httpd-24:latest=MIRROR_REGISTRY_PLACEHOLDER/rhel8/httpd
 
 }
 
+function check_mirror_registry () {
+    run_command "oc adm new-project sample-test"
+    ret=0
+    run_command "oc import-image mytestimage --from=quay.io/openshifttest/busybox@sha256:c5439d7db88ab5423999530349d327b04279ad3161d7596d2126dfb5b02bfd1f --confirm -n sample-test" || ret=$?
```
Comment (jianlinliu) [inline ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:119]: s/||/;/ ?

Comment (jianzhangbjz) [reply ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:119]: I guess no need it.

Comment (jianlinliu) [reply ci-operator/step-registry/set-sample-operator/disconnected/set-sample-operator-disconnected-commands.sh:119]: Why?
Per the above logic, only when oc command failed, ret will be assigned, if that, the following if block sounds like missing something.

**Comment by jianzhangbjz** [comment]
All failed at, @jianlinliu Could you help have a look? Thanks!
```yaml
{  uster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
ci-op-1p0ny330-e2dd4-bastion.mirror-registry.qe.azure.devcluster.openshift.com does not take effect yet on internet, waiting...
/bin/bash: line 18: dig: command not found
!!!!!!!!!!
Something wrong, pls check your dns provider
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"k8s.io/test-infra/prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2022-08-31T07:45:22Z"}
error: failed to execute wrapped command: exit status 2
}
```

**Comment by jianlinliu** [comment]
it will be fixed in https://github.com/openshift/release/pull/31840

**Comment by jianzhangbjz** [comment]
It passed based on https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/31837/rehearse-31837-periodic-ci-openshift-openshift-tests-private-release-4.11-nightly-4.11-upgrade-from-stable-4.10-gcp-ipi-disconnected-private/1564874391588704256/artifacts/gcp-ipi-disconnected-private/set-sample-operator-disconnected/build-log.txt
@bandrade Could you help add `/lgtm` label for it? Thanks!

**Pull request merged**